data/extension/init/templates/tinygo/envoy.access_loggers/default/go.mod

```diff
@@ -1,8 +0,0 @@
-module {{ .Extension.Name }}
```

Comment:
fyi @opennota this is the file I had to nuke per https://github.com/golang/go/issues/41191#issuecomment-805398783

Will watch the build output but guessing it will fail expecting go.mod

Reply:
used a workaround to keep the file. rewrote desc with the outcome